prusti-interface/src/specs/typed.rs

```diff
@@ -49,37 +49,55 @@ pub type Trigger = common::Trigger<ExpressionId, LocalDefId>;
 pub type Pledge<'tcx> = common::Pledge<ExpressionId, LocalDefId, (mir::Local, ty::Ty<'tcx>)>;
 
 pub trait Spanned<'tcx> {
-    fn get_spans(&self, tcx: TyCtxt<'tcx>) -> Vec<Span>;
+    fn get_spans(&self, mir_body: &mir::Body<'tcx>, tcx: TyCtxt<'tcx>) -> Vec<Span>;
```

Comment:
Could you add a comment to explain the parameters?

Reply:
For both of your comments: this argument was added to be able to get spans from `mir::Local`s. A local variable is an index that only makes sense within its MIR function and the only way to get position information is to check with the MIR body. Fortunately, all places that called `get_spans` already had a reference to the MIR body. I'll add a comment.

Reply:
I see, thanks!